pylabrobot/resources/ml_star/mfx_modules.py

```diff
@@ -0,0 +1,228 @@
+""" ML Star MFX modules (including machine defintions placed on a MFX carrier) """
+
+from __future__ import annotations
+
+import logging
+from typing import List, Optional, Union
+
+from pylabrobot.resources.resource import Resource
+from pylabrobot.resources.carrier import Coordinate
+
+logger = logging.getLogger("pylabrobot")
+
+
+class MFXSite(Resource):
+  """ A single site within a MFX module. """
```

Comment:
Yes, it is exactly the same and can therefore be replaced by `CarrierSite`.

I simply chose to name it `MFXSite` to avoid confusion about why a `CarrierSite` houses a `MFXModule` which houses a `CarrierSite`?